styles.css

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+section {
+  background-image: url('img/header-illsutration-mobile.png');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center top -15px;
+  height: 100vh;
+  width: 100%;
+  padding-top: 50px; 
+}
```

Comment:
It's great that you have made a good number of the requested adjustments. Kindly consider to adjust the arrow to align with the background image as shown below:

Your design:
![image](https://user-images.githubusercontent.com/20554583/235437094-742c3c06-388e-4ddf-8646-eac6b98fedb0.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/235437128-6288e390-c874-4e83-81fe-c4dde38e1dc8.png)